akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -561,6 +561,43 @@ object Source {
       case akka.actor.Status.Success                        => CompletionStrategy.Draining
     }, { case akka.actor.Status.Failure(cause)              => cause }, bufferSize, overflowStrategy)
 
+  /**
+   * INTERNAL API
+   */
+  @InternalApi private[akka] def actorRefWithAck[T](
+      ackMessage: Any,
+      completionMatcher: PartialFunction[Any, CompletionStrategy],
+      failureMatcher: PartialFunction[Any, Throwable]): Source[T, ActorRef] = {
+    Source
+      .fromGraph(new ActorRefBackpressureSource(ackMessage, completionMatcher, failureMatcher))
+      .withAttributes(DefaultAttributes.actorRefWithAckSource)
```

Comment:
Define the default attribute directly as a default on the graph stage using `initialAttributes` instead